relayer/src/__tests__/source.test.ts

```diff
@@ -0,0 +1,56 @@
+import { U64 } from "@polkadot/types/primitive";
+import { Observable } from "rxjs";
+import Source from "../source";
+import { apiMock, loggerMock } from "../mocks";
+
+describe("Source class", () => {
+    const params = {
+        api: apiMock,
+        chain: "Random relay chain name",
+        feedId: 66 as unknown as U64,
```

Comment:
I do not think `U64` is a `number`. Even in tests, types should match instead of lying.

The same issue is present in mocks, a bunch of types do not match.